src/tests/JIT/opt/SVE/PredicateInstructions.csproj

```diff
@@ -11,5 +11,6 @@
     </Compile>
     <CLRTestEnvironmentVariable Include="DOTNET_TieredCompilation" Value="0" />
     <CLRTestEnvironmentVariable Include="DOTNET_JITMinOpts" Value="0" />
+    <CLRTestEnvironmentVariable Include="DOTNET_EnableHWIntrinsic" Value="1" />
```

Comment:
Shouldn't this be skipped when HWIntrinsics are disabled instead?

Reply:
Remember, this is the asm check test and not the functional one, which do not work in [stress scenarios](https://github.com/dotnet/runtime/blob/e16eeab38b7540bb02b85448f4c6bb66700dabbf/docs/workflow/testing/coreclr/disasm-checks.md#test-run-limitations), `DOTNET_EnableHWIntrinsic` being one of them. That reminds me that I should update that doc to call it out.